index.php

```diff
@@ -4,42 +4,45 @@
-    (updated November 27, 2018).</li>
-  <li> <a href="api/1.2/openvx-standard-headers-1.2.tar.bz2"> OpenVX 1.2 header
-    package </a> (updated October 20, 2017).</li>
-  <li> <a href="sample/openvx_sample_1.2.tar.bz2">OpenVX 1.2 sample code</a>
-    (updated January 17, 2018). </p>
-</ul>
-
-<p>There are two KHR extensions to OpenVX that are supported by the
-  OpenVX 1.2 conformance tests: </p>
-<ul>
-  <li>OpenVX 1.2.1 Neural Network extension
-    (<a href="extensions/vx_khr_nn/1.2.1/vx_khr_nn_1_2_1.html">HTML</a>,
-    <a href="extensions/vx_khr_nn/1.2.1/vx_khr_nn_1_2_1.pdf">PDF</a>)
-    (updated August 15, 2018).</li>
-  <li>OpenVX 1.1.1 Export and Import extension
-    (<a href="extensions/vx_khr_ix/1.1.1/vx_khr_export_and_import_1_1_1.html">HTML</a>,
-    <a href="extensions/vx_khr_ix/1.1.1/vx_khr_export_and_import_1_1_1.pdf">PDF</a>)
-      (updated August 15, 2018).</li>
-    <ul>
-      <li>This extension designed for OpenVX 1.1 is valid without change for OpenVX 1.2(.1).</li>
-    </ul>
-  <li> <a href="api/1.2/openvx-extension-headers-1.2.tar.bz2"> OpenVX 1.2
-      extension header package </a> (updated November 8, 2017).
+<p>The current version of OpenVX is OpenVX 1.3.0:</p>
+  <ul>
+    <li> OpenVX 1.3.0 specification (<a href="specs/1.3/OpenVX_Specification_1_3.html">HTML</a>,
+      <a href="specs/1.3/OpenVX_Specification_1_3.pdf">PDF</a>)
+        (updated August 08, 2019).
+    </li>
+    <li> <a href="api/1.3/openvx-standard-headers-1.3.tar.bz2"> OpenVX 1.3 header package </a> 
+    (updated August 08, 2019).
+    </li>
+  </ul>
+
+<p>There are two KHR extensions to OpenVX that are supported by the OpenVX 1.3 conformance tests: </p>
+  <ul>
+    <li>OpenVX 1.3 Feature Sets extension
+      (<a href="extensions/vx_khr_feature_sets/1.0/html/vx_khr_feature_sets_1_0.html">HTML</a>,
+      <a href="extensions/vx_khr_feature_sets/1.0/vx_khr_feature_sets_1_0.pdf">PDF</a>)
+      (updated August 08, 2019).
+    </li>
+    <li>OpenVX 1.3 Neural Network extension
+      (<a href="extensions/vx_khr_nn/1.3/html/vx_khr_nn_1_3.html">HTML</a>,
+      <a href="extensions/vx_khr_nn/1.3/vx_khr_nn_1_3.pdf">PDF</a>)
+      (updated August 08, 2019).
+    </li>
+    <li>OpenVX 1.3 Import Kernels extension
+      (<a href="extensions/vx_khr_import_kernel/1.3/html/vx_khr_import_kernel_1_3.html">HTML</a>,
+      <a href="extensions/vx_khr_import_kernel/1.3/vx_khr_import_kernel_1_3.pdf">PDF</a>)
+      (updated August 08, 2019).
+    </li>
+    <li> 
+      <a href="api/1.3/openvx-extension-headers-1.3.tar.bz2"> OpenVX 1.3 extension header package </a> (updated August 08, 2019).
+    </li>
     <ul>
       <li>
-        This package contains header files for these officially supported
-        extensions <em>and</em> for the provisional extensions listed below.</li>
+        This package contains header files for these officially supported extensions <em>and</em> for the provisional extensions listed below.
+      </li>
     </ul>
 </ul>
```

Comment:
Version 1.1 of the SC spec is now "old" because it's replaced with a feature set in 1.3.  The OpenVX 1.1 SC spec should no longer be called "current" and it should move to the "older specs" section.